Misc/NEWS

```diff
@@ -66,6 +66,19 @@ Extension Modules
 Library
 -------
 
+- bpo-29532: Altering a kwarg dictionary passed to functools.partial()
+  no longer affects a partial object after creation.
+
+- bpo-22807: Add uuid.SafeUUID and uuid.UUID.is_safe to relay information from
```

Comment:
22807, 29576 and 29534 were master-only changes and they shouldn't be listed in 3.6 branch.